### PR TITLE
feat: add tags via tags flag

### PR DESCRIPTION
**Pull request by Monforton**
Addresses #2. Also this de-minifies the test json files so they are more easily editable.

### CODE REVIEW

File: pkg/client/client_test.go [hunk 1]
```diff
@@ -20,6 +21,7 @@ func TestSendReports(t *testing.T) {
 				fernUrl:     mockFernReporter.URL,
 				projectName: testProjectName,
 				filePattern: reportsCombinedPattern,
+				tags:        "test,tagtest,9=-+_",
```
Comment (eolatham) [inline pkg/client/client_test.go]: Could we make this a variable/constant, like `testProjectName`?

File: pkg/client/parse.go [hunk 2]
```diff
@@ -166,3 +167,10 @@ func getEndTime(startTime time.Time, durationSeconds string) (endTime time.Time,
 	endTime = startTime.Add(ms)
 	return
 }
+
+func convertToTags(tagString string) (tags []fern.Tag) {
```
Comment (eolatham) [inline pkg/client/parse.go:171]: This is implicitly tested by other functions, but we should probably add a small unit test for this function anyway.

**Review by eolatham — COMMENTED** [review]
Good change, thanks for formatting the static files too. I'm going to create a separate issue to document/automate the process of generating those files.

File: pkg/client/client_test.go (hunk 1 above)
Comment (Monforton) [reply pkg/client/client_test.go]: Moved to `exampleTags`

File: pkg/client/parse.go (hunk 2 above)
Comment (Monforton) [reply pkg/client/parse.go:171]: added unit test for `convertToTags`

**Review by eolatham — APPROVED** [review]

**Pull request merged**